targetplatform/r2021-03.tpd

```diff
@@ -0,0 +1,23 @@
+target "2021-03 Release" with source,requirements
+location "http://download.eclipse.org/releases/2021-03/" {
+	org.eclipse.equinox.p2.discovery.feature.feature.group
+	org.eclipse.equinox.executable.feature.group
+	org.eclipse.emf.sdk.feature.group
+}
+
+location "http://ghillairet.github.io/p2" {
```

Comment:
@planger didn't we publish a version of emfjson since we moved it to eclipse?

Reply:
No we didn't publish an emfjson release on the Eclipse infra yet :/
I think we'll have to stick with this for now here.